src/controllers/userRequestHistory.js

```diff
@@ -0,0 +1,67 @@
+import models from '../models';
```

Comment:
Can we have this controller inside the Trips.js file instead? That file contains the methods to create trips so it'll be better if we have the method to get user trips in it also.